apps/user/schema.py

```diff
@@ -84,6 +85,15 @@ class Meta:
     def resolve_display_picture(root, info, **kwargs) -> Union[str, None]:
         return root.profile.display_picture_id
 
+    @staticmethod
+    @only_me
+    def resolve_display_picture_url(root, info, **kwargs) -> Union[str, None]:
+        display_picture = root.profile.display_picture.file
+        if display_picture:
+            return info.context.request.build_absolute_uri(
+                URLCachedFileField.name_to_representation(display_picture)
+            )
```

Comment:
This should be enough for now.

```suggestion
        return info.context.dl.user.display_picture.load(root.id)
```